lib/fog/sakuracloud/models/volume/disk.rb

```diff
@@ -38,6 +38,11 @@ def carve_hostname_on_disk(hostname)
           true
         end
 
+        def attach(server_id)
+          service.attach_disk(id, server_id)
+          true
```

Comment:
Yeah. Makes sense. It would be annoying if the request failed and it returned true. There isn't an easy way to check the result of the request so this is bad.

And i agree we should do this in another PR.

EDIT: Maybe you could add a test around it?

@sawanoboly Please merge this as soon as you think it is good to go.

Thanks!